DirectProgramming/DPC++FPGA/Tutorials/Tools/board_test/README.md

```diff
@@ -0,0 +1,346 @@
+
+3. **Host to kernel interface:** The test ensures the host to kernel communication is correct and that the host can launch a kernel successfully. It also measures the roundtrip kernel launch latency and throughput (number of kernels/ms) of single task no-operation kernels.
+
+In addition to the above interface checks, the test also measures the **Kernel clock frequency**. The test measures the frequency that the programmed kernel is running at on the FPGA device and reports it. By default, this test fails if the measured frequency is not within 2% of the compiled frequency.<br>
+*Note:* The test allows overriding this failure, however, this is may lead to functional errors and is not recommended. The override option is provided to allow debug in case where platform design changes are done to force kernel to run at slower clock (this is not a common use-case). Usage section describes how to override this option if required.
+
+The complete test is divided into six sub-tests. By default, all tests are run. You can choose to run a single test using `-test=<test number>` option. Please see [Running the Sample](#running-the-sample) for test usage instructions.
+1. Host Speed and Host Read Write Test
+2. Kernel Clock Frequency Test
+3. Kernel Launch Test
+4. Kernel Latency Measurement
+5. Kernel-to-Memory Read Write Test
+6. Kernel-to-Memory Bandwidth Test<br>
+*Note:* It is recommended to run all tests at least once to ensure that the platform interfaces are fully functional.
+
+## Key Concepts
+* Sample to demonstrate platform interface checks
+
+## License  
+Code samples are licensed under the MIT license. See
+[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+
+## Building the `board_test` code sample
+
+### Include Files
+The included header `dpc_common.hpp` is located at `$ONEAPI_ROOT/dev-utilities/latest/include` on your development system.
+
+### Running Samples in DevCloud
+If running a sample in the Intel DevCloud, remember that you must specify the type of compute node and whether to run in batch or interactive mode. Compiles to FPGA are only supported on fpga_compile nodes. Executing programs on FPGA hardware is only supported on fpga_runtime nodes of the appropriate type, such as fpga_runtime:arria10 or fpga_runtime:stratix10.  Neither compiling nor executing programs on FPGA hardware are supported on the login nodes. For more information, see the [Intel® oneAPI Base Toolkit Get Started Guide](https://devcloud.intel.com/oneapi/documentation/base-toolkit/).
+
+When compiling for FPGA hardware, it is recommended to increase the job timeout to 12h.
+

```

Comment:
If you are planning on adding Windows support, you'll have to add a section on how to compile on Windows

Reply:
Added a section on Windows, will be in next upload